docs/api/cli.md

```diff
@@ -167,4 +167,4 @@ This command will show the current version of Tauri.
 
 ## CLI usage
 
-See more about the usage through this [complete guide](/docs/development/development-cycle).
+See more about the usage through this [complete guide](/docs/manual/02-Development/Development Cycle).
```

Comment:
Not a fan in cases like this where the numbers are hard-coded in the URL

Reply:
They are actually not, see this https://docusaurus.io/docs/sidebar/autogenerated#using-number-prefixes

They get stripped when compiling the mdx, so they are just for development. The numbers just tell docusaurus which order the categories should have without needed an extra config file